src/Jackett.Common/Definitions/nekobt.yml

```
@@ -0,0 +1,250 @@
+---
+id: nekobt
+name: nekoBT
+description: "nekoBT is a Private Anime Torrent Tracker specializing in anime content with subtitle and fansub filtering"
+language: en-US
+type: public
+encoding: UTF-8
+links:
+  - https://nekobt.to/
+
+caps:
+  categorymappings:
+    - {id: 5000, cat: TV, desc: "TV"}
+    - {id: 5070, cat: TV/Anime, desc: "Anime"}
+    - {id: 2000, cat: Movies, desc: "Movies"}
```

```suggestion
    - {id: 2000, cat: Movies/Other, desc: "Movies"}
```

Just so it's line with other indexers, distinguishing anime movies from others